Cargo.toml

```diff
@@ -26,6 +27,13 @@ serde_json = "1.0.139"
 tket2 = "0.7.2"
 tket2-hseries = { version = "0.9.0", features = ["cli"] }
 
+[patch.crates-io]
+hugr = { git = "https://github.com/CQCL/hugr", branch = "acl/inline_call" }
```

Comment:
Do we want to point the that branch or maybe do we want to make a new one named something like hugr-qir?